### PR TITLE
Add CJS bundle

### LINKED ISSUE #40: Distribution in CJS / UMD format
Hi Matthew,

Following up on [the Twitter conversation](https://twitter.com/danburzo/status/1175457092007862272), I'm opening this issue about packaging for use in Node and (older) browsers.

I can add a Rollup config to build CJS / UMD bundles in a PR, but I noticed the use of `"type": "module"` in package.json ([TIL!](https://medium.com/@nodejs/announcing-a-new-experimental-modules-1be8d2d6c2ff)), and I'm not sure how making the `main` field point to CJS and `module` point to ES can impact people's setups...

### PR DESCRIPTION
**Pull request by danburzo**
Added Rollup as a build tool with CJS as the output format.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,18 +2,20 @@
...
     "debug.js",
     "machine.js"
   ],
   "scripts": {
     "minify": "terser machine.js -o machine.min.js",
     "bundlesize": "npm run minify && bundlesize --config bundlesize.json",
     "server": "http-server -p 1965",
-    "test": "node-qunit-puppeteer http://localhost:1965/test/test.html 10000"
+    "test": "node-qunit-puppeteer http://localhost:1965/test/test.html 10000",
+    "build": "rollup -d dist -f cjs machine.js debug.js"
```
Comment (matthewp) [inline package.json]: Would you add this as well:

```suggestion
    "build": "rollup -d dist -f cjs machine.js debug.js",
    "preversion": "npm run build"
```

**Comment by matthewp** [comment]
One small change suggested, otherwise looks good!

**Comment by danburzo** [comment]
Done!

**Comment by matthewp** [comment]
Thanks!

**Pull request merged**